ansible/roles/sap-hana-scaleout-standby/tasks/storage.yml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
This is pretty much a copy of the `storage` role. It should be usable with some small modifications right?

Reply:
Done. Updated the code to use the common storage role.